pitop/processing/utils/vision_functions.py

```diff
@@ -66,9 +66,34 @@ def find_largest_rectangle(rectangles):
     return rectangles[largest_index]
 
 
-def scale_frame(frame, scale):
-    cv2 = import_opencv()
-    scaled_width = int(frame.shape[1] * scale)
-    scaled_height = int(frame.shape[0] * scale)
-    dim = (scaled_width, scaled_height)
-    return cv2.resize(frame, dim, interpolation=cv2.INTER_AREA)
+def center_reposition(center, frame):
+    """Reposition center so that (0, 0) is in the middle of the frame instead
+    of OpenCV standard which is at top left.
+
+    :param center: OpenCV center (x, y)
+    :param frame: Frame to reposition center within
+    :return:
+    """
+    if center is None:
+        return None
+    # convert so (0, 0) is at the middle bottom of the frame
+    center_x = center[0] - int(frame.shape[1] / 2)
+    center_y = int(frame.shape[0] / 2) - center[1]
+
+    return center_x, center_y
+
+
+def get_control_angle(center, frame):
```

Comment:
Totally not obvious what this 'vision function' is doing, and why anyone would use it, or not - aside from the actual place where it's used...

Reply:
Added description and tried to make it clearer